src/browser/process.rs

```diff
@@ -233,6 +238,15 @@ impl Process {
 
         args.extend(&DEFAULT_ARGS);
 
+        if !launch_options.args.is_empty() {
+            let extra_args: Vec<&str> = launch_options
+                .args
+                .iter()
+                .map(|a| a.to_str().unwrap())
```

Comment:
Could `flat_map` this to get rid of the `unwrap()`

Reply:
You probably don't even need to `collect`. You can just pass the iterator to `extend`